ZuluContent/Zulu/Engines/Loot/LootTable.cs

```diff
@@ -100,5 +111,18 @@ public partial class LootTable
             {Jewelry, 1, 10, 0.5},
             {Clothing, 1, 10, 0.5},
         };
+
+        // level6map, golden dragon, behemoth, balron, arachnidqueen
+        public static readonly LootTable Table2 = new LootTable(itemLevel: 10, itemChance: 1.0)
```

Comment:
This is definitely not final, was just messing around testing some stuff here

Reply:
All the loot generation code is pretty much a [straight port from the ZHA scripts](https://github.com/ZuluHotelAustralia/zha/blob/master/scripts/include/starteqp.inc#L47). I can't think of any reason 10+ levels wouldn't work, but that said the [npcdesc from ZHA](https://github.com/ZuluHotelAustralia/zha/blob/master/config/starteqp.cfg) doesn't include any npc entries greater than 10 which might not be a coincidence.

Reply:
I took some time to look into this yesterday and the main reasons were incorrect math in the loot generation code (I found a case where whoever ported it used + instead of * when comparing it to the POL ZH reference).

The other reason was that you only get a bonus to enchantmentlevel when the itemLevel is 5, so the further you move away from that the less likely it is to get good loot